src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer;
+
+import com.google.api.generator.engine.ast.AssignmentExpr;
+import com.google.api.generator.engine.ast.CommentStatement;
+import com.google.api.generator.engine.ast.LineComment;
+import com.google.api.generator.engine.ast.MethodInvocationExpr;
+import com.google.api.generator.engine.ast.TryCatchStatement;
+import com.google.api.generator.engine.ast.TypeNode;
+import com.google.api.generator.engine.ast.Variable;
+import com.google.api.generator.engine.ast.VariableExpr;
+import com.google.api.generator.gapic.model.Method;
+import com.google.api.generator.gapic.model.MethodArgument;
+import com.google.api.generator.gapic.model.ResourceName;
+import com.google.api.generator.gapic.utils.JavaStyle;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public final class SampleCodeHelperComposer {
+
+  public static TryCatchStatement composeRpcMethodSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    // Paged Unary RPC method.
+    if (method.isPaged()) {
+      return composePagedUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
+    }
+    // Long-running operation Unary RPC method.
+    if (method.hasLro()) {
+      return composeLroUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
+    }
+    // Pure Unary RPC method.
+    return composeUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
+  }
+
+  public static TryCatchStatement composeRpcDefaultMethodSampleCode(
```

Comment:
Could you please elaborate on the difference between this and `composeRpcMethodSampleCode`? Can we combine them?

Reply:
[line](https://github.com/googleapis/gapic-generator-java/blob/rpc_unary_sc_31/src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java#L485) `createMethodVariants` -> `composeRpcMethodSampleCode` [linke](https://github.com/googleapis/gapic-generator-java/blob/rpc_unary_sc_31/src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java#L567) 
[line](https://github.com/googleapis/gapic-generator-java/blob/rpc_unary_sc_31/src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java#L486) `createMethodDefaultMethod` -> `composeRpcDefaultMethodSampleCode` [line](https://github.com/googleapis/gapic-generator-java/blob/rpc_unary_sc_31/src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java#L641)
DefaultMethodSampleCode is responsible for the method has the pass in argument is `request`. The implementation in PR #517